### PR TITLE
ADD: Add new lines to comments, so that the database holds line breaks.

### PR DESCRIPTION
**Pull request by JRRS1982**
Changing the way text is saved, so that when you enter text as a string, with breaks,
you then get that message back as breaks, over new lines. Not just as a string.
simple_format method is a rails method that takes in formatting as arguments.

### CODE REVIEW

**Comment by JRRS1982** [comment]
[Posts can have new lines in them [SMALL]](https://trello.com/c/vEVLKcSf/11-posts-can-have-new-lines-in-them-small)

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -98,6 +98,10 @@ input[type=text], input[type=password], input[type=email] {
   border-radius: 5px;
 }
 
+textarea {
```
Comment (JRRS1982) [inline app/assets/stylesheets/application.css:101]: Agreed, we had this originally in the html, and were going to look at the css later. Nice and quick!!

**Review by JRRS1982 — COMMENTED** [review]
Think its fine, thoughts?

**Comment by Hives** [comment]
Let's get a feature test or two on this feature before we merge it

**Comment by arthurashman** [comment]
Looks good.

**Pull request merged**